src/main/java/com/wultra/security/userdatastore/userclaims/UserClaimsService.java

```diff
@@ -15,21 +15,33 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package com.wultra.security.userdatastore;
+package com.wultra.security.userdatastore.userclaims;
```

Comment:
ℹ️ Just highlight, that I suggest using a kind of _Entity-Control-Boundary_ pattern and do not slice packages by layer such as `service`, `dao` etc. Of course, when we introduce a REST client, we may extract the model and so on.